bin/start_gke

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This is to be used by the Makefile for a start gke target.
+
+set -eo pipefail
+
+MY_IP=`curl icanhazip.com`
+echo "local ip is $MY_IP"
+
+USERNAME=`gcloud config get-value account | awk -F"@" '{print $1}'`
+CLUSTERNAME=$USERNAME-gke-sandbox
+
+# Create GKE Cluster - No TTL available, will need a external monitor with cleanup
+gcloud config set project moz-fx-data-gke-sandbox
```

Comment:
You may want to set the project back at the end of the script, or use an explicit `--project` flag with the gcloud tool.

Reply:
I agree we should probably use `--project` explicitly instead of changing the user's general setting. Another option (that needs to be careful about error handling) is to get the current project before setting it to the gke and project and to reset the project after the script runs.